bug-bounty-list/bug-bounty-list.json

```diff
@@ -3889,6 +3889,16 @@
     "hall_of_fame": true,
     "safe_harbor": "partial"
   },
+  {
+    "program_name": "Livestream",
+    "policy_url": "https://hackerone.com/livestream",
```

Comment:
This appears to be a private program – note that the program has to be public (its terms and submission URL) to qualify for this list. Can you remove this entry?